### PR TITLE
feat: add schema.json doc splitter

### PR DESCRIPTION
**Pull request by thalesmg**
[EMQX-8330](https://emqx.atlassian.net/browse/EMQX-8330)

Adds a script to split the `schema.json` file generated by EMQX's build into smaller subsections for better documentation navigation.

- [x] sub-dirs, one for .json files and one for .md files
- [x] Add a index.md and put the preamble there
- [x] Cross-doc refs and anchors 
- [x] Font size in others.md
- [x] broker config should be in emqx.md (not in others.md)
- [x] copy hocon_md.erl and hocon_schema_md.erl to local
- [x] tabular view for paths and env overrides
- [x] experiment with embedded html tables for field attributes (name, type, default, description)

### Related PRs

Those will need to be merged after the changes here are approved and stabilized.

- https://github.com/emqx/emqx/pull/9485
- https://github.com/emqx/hocon/pull/214

### CODE REVIEW

File: dist/emqx-enterprise/config-license.json [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+    "desc": "License provisioned as a string.",
+    "fields": [
+      {
+        "default": {
+          "hocon": "\"MjIwMTExCjAKMTAKRXZhbHVhdGlvbgpjb250YWN0QGVtcXguaW8KZGVmYXVsdAoyMDIyMDQxOQoxODI1CjEwMDAK.MEQCICbgRVijCQov2hrvZXR1mk9Oa+tyV1F5oJ6iOZeSHjnQAiB9dUiVeaZekDOjztk+NCWjhk4PG8tWfw2uFZWruSzD6g==\"",
+          "oneliner": true
+        },
+        "desc": "License string",
+        "name": "key",
+        "raw_default": [
```
Comment (zmstone) [inline dist/emqx-enterprise/config-license.json]: this looks wrong. maybe binary() should be used in EMQX.

**Comment by zmstone** [comment]
can we render the markdown for review in this PR ?

**Comment by zmstone** [comment]
there is maybe no need to have the config- prefix for json files.

File: dist/emqx-enterprise/config-license.json (hunk 1 above)
Comment (thalesmg) [reply dist/emqx-enterprise/config-license.json]: :heavy_check_mark:

File: preamble.en.md [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+    "DHE-DSS-AES128-SHA",
+    "ECDH-ECDSA-AES128-SHA",
+    "ECDH-RSA-AES128-SHA"
+  ]
+```
+
+For PSK enabled listeners
+
+```
+ciphers =
```
Comment (zmstone) [inline preamble.en.md:311]: there were a few PSK ciphers added recently. please sync it from emqx.git master branch.

Comment (thalesmg) [reply preamble.en.md:311]: :heavy_check_mark:

**Review by ieQu1 — APPROVED** [review]

**Review by zmstone — APPROVED** [review]

**Pull request merged**